zerver/tests/test_import_export.py

```diff
@@ -390,7 +392,8 @@ def check_variable_type(user_profile_id: int, realm_id: int) -> None:
 
         # Test avatars
         fn = os.path.join(full_data['avatar_dir'], original_avatar_path_id)
-        fn_data = open(fn, 'rb').read()
+        with open(fn, 'rb') as file:
+            fn_data = file.read()
```

Comment:
We should call this `f`, not `file`, which is a keyword.

Reply:
It’s actually just a Python 2–only builtin, although it’s a bit irritating that it still tends to get syntax-highlighted that way.

Reply:
Ah, gotcha. The reason I used `file` was actually because of mypy apparently getting upset about reusing `f`.